tools/bpf2c/bpf_code_generator.cpp

```diff
@@ -646,9 +661,47 @@ bpf_code_generator::emit_c_code(std::ostream& output_stream)
     }
 
     for (auto& [name, section] : sections) {
-        auto function_name = !section.function_name.empty() ? section.function_name : name;
+        auto program_name = !section.program_name.empty() ? section.program_name : name;
+
+        // Emit the program and attach type GUID.
+        std::string program_type_name = program_name + "_program_type_guid";
+        std::string attach_type_name = program_name + "_attach_type_guid";
+
+        output_stream << format_string(
+                             "static GUID %s = %s;",
+                             sanitize_name(program_type_name),
+                             format_guid(&section.program_type))
+                      << std::endl;
+        output_stream << format_string(
+                             "static GUID %s = %s;",
```

Comment:
```suggestion
                             "static ebpf_attach_type_t %s = %s;",
```

Reply:
bpf_code_generator.cpp is compiled on Linux also (as part of bpf2c-test in CI/CD). So this file cannot include Windows specific headers. Including ebpfapi.h to this file causes issues.